tests/cpptests/test_cpp_index.cpp

```diff
@@ -1600,23 +1600,24 @@ TEST_F(IndexTest, testRawDocId) {
   size_t index_memsize = 0;
   InvertedIndex *idx = NewInvertedIndex(Index_DocIdsOnly, 1, &index_memsize);
   IndexEncoder enc = InvertedIndex_GetEncoder(idx->flags);
+  constexpr size_t n_ids = INDEX_BLOCK_SIZE_DOCID_ONLY * 3;
 
   // Add a few entries, all with an odd docId
-  for (t_docId id = 1; id < INDEX_BLOCK_SIZE; id += 2) {
+  for (t_docId id = 1; id < n_ids; id += 2) {
     InvertedIndex_WriteEntryGeneric(idx, enc, id, NULL);
   }
 
   // Test that we can read them back
   IndexReader *ir = NewTermIndexReader(idx, NULL, RS_FIELDMASK_ALL, NULL, 1);
   RSIndexResult *cur;
-  for (t_docId id = 1; id < INDEX_BLOCK_SIZE; id += 2) {
+  for (t_docId id = 1; id < n_ids; id += 2) {
     ASSERT_EQ(INDEXREAD_OK, IR_Read(ir, &cur));
     ASSERT_EQ(id, cur->docId);
   }
   ASSERT_EQ(INDEXREAD_EOF, IR_Read(ir, &cur));
 
   // Test that we can skip to all the ids
-  for (t_docId id = 1; id < INDEX_BLOCK_SIZE; id++) {
+  for (t_docId id = 2000; id < n_ids; id++) {
```

Comment:
Why from `2000`?

Reply:
oops. I had to drop part of the revert because skipping to 2000 returned id 3. this is a leftover